src/wwwroot/template-editor/adapterService.js

```diff
@@ -38,8 +41,32 @@
 
     function init() {
       langKey = getPreferredLanguage();
+      decodedToken = getToken();
     };
 
+    function getToken() {
+      var token = $window.localStorage.getItem('jwtToken');
+      if (!token) {
+        logOut();
+      } else if (token != encodedToken) {
+        encodedToken = token;
+        return jwtHelper.decodeToken(encodedToken);
+      }
+    }
+
+    function logOut() {
```

Comment:
What do you think about delegating this kind of things to WebApp? I mean, redirecting or something like that (in order of keeping it simple)

Reply:
Good idea 🔢